release.sh

```diff
@@ -16,13 +16,13 @@ git add 'ipwb/'$FILE_NAME
 git commit -m "RELEASE: Bump version to "$VERSION_STRING
 
 # Create a tag in repo
-TAG_NAME='v'$VERSION_STRING
+TAG_NAME='Release '$VERSION_STRING
 git tag $TAG_NAME
 git push
 git push origin $TAG_NAME
```

Comment:
We don't want spaces in a tag name:

```suggestion
git tag $VERSION_STRING
git push
git push origin $VERSION_STRING
```